app/graphql/mutations/create_saved_search.rb

```diff
@@ -31,6 +31,15 @@ def build_name_label(search_info)
               result = result + "#{search_term}|"
               entries = entries + 1 
             end 
+            if search_info["crowd_agg_filters"].present?
```

Comment:
Might make sense to move this block after the aggs so naming is consistent with what user sees on screen